ci-operator/config/openshift/hypershift/openshift-hypershift-main.yaml

```diff
@@ -38,6 +38,28 @@ promotion:
   name: "4.14"
   namespace: ocp
 releases:
+  hosted-ocp-4-12-latest:
+    candidate:
+      product: ocp
+      stream: ci
+      version: "4.12"
+  hosted-ocp-4-12-previous:
+    candidate:
+      product: ocp
+      relative: 1
+      stream: ci
+      version: "4.12"
+  hosted-ocp-4-13-latest:
+    candidate:
+      product: ocp
+      stream: ci
+      version: "4.13"
+  hosted-ocp-4-13-previous:
+    candidate:
+      product: ocp
+      relative: 1
+      stream: ci
+      version: "4.13"
```

Comment:
We are going to have to update and possible expand this as more releases get branches right? Is this sustainable?  For example, when ROSA start moving to 4.14, what does this look like?

Reply:
If I am not mistaken, hypershift operator only supports N-2 versions of OCP, right? so we would only need to update the versions here to `4.14` and `4.13` instead of `4.13` and `4.12` and we won't have to expand this any further.